### ISSUE TITLE
0075-feel-exponent

### ISSUE THREAD

**Issue by saig0** [camunda/tck#14]
```
Failed to parse FEEL expression '(function() "foo") ** 4': failed to parse expression '(function() "foo") ** 4': Expected (StringIn("<=", ">=", "<", ">", "!=", "=") | "between" | "in" | "and" | "or" | ".."):1:18, found ") ** 4"
```

**Comment by saig0** [comment]
related to https://github.com/camunda/feel-scala/issues/312

**Comment by saig0** [comment]
Resolved in DMN-Scala version `1.7.4`.
